accelerator/roles/intel/vars/main.yml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2024 Intel Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+
+intel_gaudi_device_pattern: "Processing accelerators: Habana Labs Ltd."
+
+intel_habana_packages:
+  - habanalabs-container-runtime
+  - habanalabs-dkms
```

Comment:
all other packages need to read from gaudi.json. MEanining playbook should install whatever present in Gaudi.json so that if there is any change in package later it will automatically apply just by updating Gaudi.json file

Reply:
I see amd defines the package names in role too. But I think it is a good idea to define package list only in json, to achieve centralized configuration. But for provisioning postscripts, still need to manage their own copy.

Reply:
We agree to merge and will be handled with another PR, @yupengzh-intel please add a comment TODO: move to a central config file